stable/drupal/README.md

```diff
@@ -88,9 +90,17 @@ $ helm install --name my-release -f values.yaml stable/drupal
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
-## Persistence
+## Image
```

Comment:
You're right. I was trying to make a header that would encompass subheaders for both the existing Persistence header and this new volume mounts header - and thought Image might make sense. But maybe you're right. Let me remove that extra image header hierarchy, and see if it seems less redundant?